Library/DensityCDF.Rmd

```diff
@@ -24,22 +24,33 @@ pacman::p_load(packages, character.only = TRUE, install = FALSE)
 # Load an example dataset ---------------
 data <- read_dta("https://github.com/worldbank/r-econ-visual-library/raw/master/Library/Data/DensityCDF.dta")
 
+# Printing the first five rows of the data to visualize its structure before data wrangling
+head(data)
+
+# Data wrangling ---------------
 data <- data %>%
   mutate(post = as.factor(post)) %>%
   group_by(post) %>%
   mutate(mean_revenue = mean(revenue)) %>%
   ungroup()
 
+# Printing the first five rows of the data to visualize its structure after data wrangling
+head(data)
+
+# Create a graph ---------------
 ggplot(data, aes(x = revenue, color = post)) +
   geom_vline(
     aes(xintercept = mean_revenue, color = post), 
     linetype = "dashed", size = 0.3
     ) +
-  stat_ecdf() +
+  stat_ecdf() + 
+  # stat_ecdf for empirical cumulative density function
```

Comment:
```suggestion
  # Add empirical cumulative density function
  stat_ecdf() + 

```

Reply:
Comments usually come before the line of code that implements what is being described by the comment, or in the same line as the code if after the code. This is just a convention, but because this is the common practice, adding comments in the next line seems a bit counter-intuitive, and makes me think that the comment refers to the line coming after it.